topiary-core/src/error.rs

```diff
@@ -188,3 +201,49 @@ impl From<topiary_tree_sitter_facade::ParserError> for FormatterError {
         Self::Internal("Error while parsing".into(), Some(Box::new(e)))
     }
 }
+
+impl From<NodeSpan> for FormatterError {
+    fn from(span: NodeSpan) -> Self {
+        Self::Parsing(Box::new(span))
+    }
+}
+
+#[derive(Diagnostic, Debug)]
+struct ErrorSpan {
+    #[source_code]
+    src: NamedSource<String>,
+    #[label("(ERROR) node")]
+    span: SourceSpan,
+    range: Range,
+}
+
+impl std::fmt::Display for ErrorSpan {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let start = self.range.start_point();
+        let end = self.range.end_point();
+        write!(
+            f,
+            "Parsing error between line {}, column {} and line {}, column {}",
```

Comment:
Curious if there's any way to disable the logging prefix just for `Error` span to add readability to the output:
`[YYYY-MM-DDT00:00:00Z ERROR topiary]   × Parsing error between line ...`

Reply:
Not that I'm aware of, at least with the log handler we're using. Personally, I find it useful, even if it does look a little clumsy; I could imagine `grep`ing for `ERROR`, for example.